apps/frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -41,11 +50,18 @@ const ProtectedRoute = ({
 
 	const hasRequiredPermission = checkHasPermission(
 		routePermissions,
-		authenticatedUser.groups.flatMap((group) => group.permissions),
+		userPermissions,
 	);
 
 	if (!hasRequiredPermission) {
-		return <NotFound />;
+		const FIRST_ITEM_INDEX = 0;
+
+		const firstAvailableRoute =
+			getPermittedNavigationItems(SIDEBAR_ITEMS, userPermissions)[
+				FIRST_ITEM_INDEX
+			]?.href || AppRoute.ROOT;
+
+		return <Navigate replace to={firstAvailableRoute} />;
```

Comment:
we can simplify this much and make more readable
```suggestion
		const [navigationItem] = getPermittedNavigationItems(SIDEBAR_ITEMS, userPermissions);
		const redirectLink = navigationItem?.href ?? AppRoute.ROOT;

		return <Navigate replace to={redirectLink} />;
```